dashboard/app/views/levels/editors/fields/_free_response_fields.html.haml

```diff
@@ -18,6 +18,10 @@
     = f.label :solution
     %p Optional solution markdown to display to teachers.
     = f.text_area :solution, rows: 3, class: "input-block-level"
+  .field
+    = f.label :num_attempts_allowed, "Number of Attempts Allowed"
+    %p Number of attempts students are allowed. Default is one for contained levels and infinite for standalone levels. It will not apply for levels in level groups.
```

Comment:
@dju90 does this sound right to you? This restriction will also apply to `Match` / `Multi` levels but I couldn't find a nice way to communicate that :(